cpplib/src/Python/PythonInterface.cpp

```
@@ -849,6 +854,75 @@ extern "C" {
+
+		cluster_detail::UnitCellBuilder ucb(symmvec);
+		auto buildresult = ucb.build(all.points, all.types);
+
+		geometry::SpatialGrid<FloatingPointType> space;
+		space.build(buildresult.atoms.points, cell, cutoff);
+		auto bonds = WITH_LOG_M(space, get_bonds, false);
+
+		Diagram diag(buildresult.atoms.points, bonds, cell, bools);
+		auto ce = diag.extractCells();
+
+		cpplib::voronoi::VoronoiFused vf(ce, cell.fracToCart());
```

Clang (14.0.6)

[warning] variable name 'vf' is too short, expected at least 3 characters

(readability-identifier-length)